src/cli/kfc.ts

```diff
@@ -38,7 +38,7 @@ export default function (program: RootCmd) {
           stdio: "inherit",
         });
       } catch (e) {
-        console.error(e.message);
+        console.error(`Error creating CRD generated class: ${e.message}`);
```

Comment:
```suggestion
        console.error(e, `Error creating CRD generated class`);
```

Reply:
All messages updated to display complete `e`